index.html

```diff
@@ -285,107 +271,211 @@ <h2>
+            if the battery is discharging, and set to `true` if the battery is
+            charging, the implementation is unable to report the state, or
+            there is no battery attached to the system, or otherwise.
+          </p>
+          <p>
+            When the system battery's charging state changes, the user agent
+            must run the <a>update the battery status and notify</a> algorithm
+            with {{BatteryManager/[[Charging]]}}, `true` or `false` depending
+            on whether the battery is charging or discharging, and
+            "`chargingchange`".
+          </p>
+        </section>
+        <section>
+          <h4>
+            The `[[ChargingTime]]` internal slot
+          </h4>
+          <p>
+            The {{BatteryManager/[[ChargingTime]]}} internal slot represents
+            the time remaining in seconds until the system's battery is fully
+            charged. It MUST be set to 0 if the battery is full or there is no
+            battery attached to the system, and to the value positive Infinity
+            if the battery is discharging, the implementation is unable to
+            report the remaining charging time, or otherwise.
+          </p>
+          <p>
+            When the battery charging time is updated, the user agent must run
+            the <a>update the battery status and notify</a> algorithm with
+            {{BatteryManager/[[ChargingTime]]}}, the new charging time in
+            seconds, and "`chargingtimechange`".
+          </p>
+        </section>
+        <section>
+          <h4>
+            The `[[DischargingTime]]` internal slot
+          </h4>
+          <p>
+            The {{BatteryManager/[[DischargingTime]]}} attribute represents the
+            time remaining in seconds until the system's battery is completely
+            discharged and the system is about to be suspended. It MUST be set
+            to the value positive Infinity if the battery is charging, the
+            implementation is unable to report the remaining discharging time,
+            there is no battery attached to the system, or otherwise.
+          </p>
+          <p>
+            When the battery discharging time is updated, the user agent must
+            run the <a>update the battery status and notify</a> algorithm with
+            {{BatteryManager/[[DischargingTime]]}}, the new discharging time in
+            seconds, and "`dischargingtimechange`".
+          </p>
+        </section>
+        <section>
+          <h4>
+            The `[[Level]]` internal slot
+          </h4>
+          <p>
+            The {{BatteryManager/[[Level]]}} internal slot represents the level
+            of the system's battery. It MUST be set to 0 if the system's
+            battery is depleted and the system is about to be suspended, and to
+            1.0 if the battery is full, the implementation is unable to report
```

Comment:
Don't need to fix this now, but this is a bit confusing... it doesn't say when it must be set to 0 or 1? I guess when it's initialized... but this should really be part of some initialization steps.

Reply:
Or, when initialized, run "update the battery status and notify" and set them there.

Reply:
yeah, let's do that as followup... fwiw, I'm imagining this prose need to go into some initializer steps... as an example:

https://w3c.github.io/geolocation-api/#constructing-a-geolocationposition

Reply:
Thanks, I've filed #48 to track this.